### PR TITLE
render: remove --config-override-file default

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: cmd/cluster-kube-controller-manager-operator/render/render.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func NewRenderCommand() *cobra.Command {
 	cmd.Flags().StringVar(&renderOpts.templatesDir, "templates-input-dir", "/usr/share/bootkube/manifests", "A path to a directory with manifest templates.")
 	cmd.Flags().StringVar(&renderOpts.configOverrideFile, "config-override-file", "", "A sparse KubeControllerManagerConfig.kubecontrolplane."+
 		"config.openshift.io/v1 file (default: kube-controller-manager-config-overrides.yaml in the asset-input-dir)")
-	cmd.Flags().StringVar(&renderOpts.configOutputFile, "config-output-file", "", "Output path for the KubeControllerManagerConfig yaml file.")
+	cmd.Flags().StringVar(&renderOpts.configOverrideFile, "config-override-file", "", "An additional sparse KubeAPIConfig.kubecontrolplane.config.openshift.io/v1 file for customiziation through the installer.")
```
Comment (mfojtik) [inline cmd/cluster-kube-controller-manager-operator/render/render.go]: not KubeAPIConfig

**Comment by sttts** [comment]
Blocked by https://github.com/openshift/installer/pull/457.

**Comment by sttts** [comment]
Blocked by https://github.com/openshift/installer/pull/457.

**Pull request merged**